crates/ruff/src/rules/pylint/rules/logging.rs

```diff
@@ -0,0 +1,85 @@
+use rustpython_parser::ast::{Constant, Expr, ExprKind, Keyword};
+
+use ruff_macros::{define_violation, derive_message_formats};
+
+use crate::ast::helpers::{is_logger_candidate, SimpleCallArgs};
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::registry::{Diagnostic, Rule};
+use crate::rules::flake8_logging_format::rules::LoggingLevel;
+use crate::rules::pyflakes::cformat::CFormatSummary;
+use crate::violation::Violation;
+
+define_violation!(
+    pub struct LoggingTooFewArgs;
```

Comment:
Are you open to writing documentation for these rules? We're trying to include documentation for all new rules. You can grep for `/// ## What it does` to see some examples of the expected format.